container_files/guac/guac.yaml

```diff
@@ -34,3 +34,4 @@ interval: 5m
 db-driver: postgres
 db-address: postgres://guac:guac@postgres:5432/guac?sslmode=disable
 db-migrate: true
+db-debug: false
```

Comment:
what is this change for?

Reply:
+1 to this

Reply:
@pxp928 is a missing param in the db configuration https://github.com/guacsec/guac/blob/main/pkg/cli/store.go#L64

Reply:
hmmm looks like they are not being used. I think we should remove it for the time being if not needed. Same with `db-migrate`?

Reply:
@pxp928 `db-migrate` param is used https://github.com/guacsec/guac/blob/main/pkg/assembler/backends/ent/migrate/schema.go 
I'll remove only the `db-debug`